packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -158,25 +173,27 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
             id={ this._inputId }
             onChange={ this._onChange }
             onInput={ this._onInputChange }
-            className={ css(styles.Input, (disabled ? styles.disabled : '')) }
+            className={ classNames.input }
             type='text'
             role='spinbutton'
             aria-labelledby={ label && this._labelId }
             aria-valuenow={ value }
             aria-valuemin={ min && String(min) }
             aria-valuemax={ max && String(max) }
-            onBlur={ this._validate }
+            onBlur={ this._onBlur }
             ref={ this._resolveRef('_input') }
-            onFocus={ this.focus }
+            onFocus={ this._onFocus }
             onKeyDown={ this._handleKeyDown }
             onKeyUp={ this._handleKeyUp }
             readOnly={ disabled }
+            disabled={ disabled }
             aria-disabled={ disabled }
           />
-          <span className={ styles.ArrowBox }>
+          <span className={ classNames.arrowBox }>
             <IconButton
-              className={ css('ms-UpButton', styles.UpButton, (keyboardSpinDirection === KeyboardSpinDirection.up ? styles.active : '')) }
-              styles={ this._arrowButtonStyle }
+              styles={ this._getUpArrowButtonStyles() }
+              className={ 'ms-UpButton' }
+              checked={ keyboardSpinDirection === KeyboardSpinDirection.up ? true : false }
```

Comment:
You can remove the ternary when setting checked  it can be just keyboardSpinDirection === KeyboardSpinDirection.up